cmd/net-rpc-client.go

```diff
@@ -104,7 +115,12 @@ func (rpcClient *RPCClient) dialRPCClient() (*rpc.Client, error) {
 	if err == nil && resp.Status == "200 Connected to Go RPC" {
```

Comment:
The problem is conn.Close() cannot be handled properly.  This code is directly stolen from Go's own codebase.